libs/langchain-community/src/vectorstores/zep_cloud.ts

```diff
@@ -0,0 +1,347 @@
+import { ZepClient } from "@getzep/zep-cloud";
```

Comment:
I don't think this makes sense as a vectorstore - why not just use the retriever interface?

Reply:
So the retriever is used for memory (if you want to search for messages or summaries from the conversation). The VectorStore is a class used to interact with zep collections that you can add documents to (not conversations), and then search those collections for rag use cases in the chains.

Retriever searches memory, Vector Store adds documents to collections and searches those documents. 

We have a Zep Vector Store in Python langchain sdk as well.